Packs/AzureStorageTable/README.md

```diff
@@ -0,0 +1,2 @@
+# content-azure-storage-table
+Azure Storage Table integration
```

Comment:
@guykeller 
As you know, when an empty file is provided, the CI-CD process throws an error.
I talked about it with Bar, and it was agreed that it would be resolved by Bar later in an internal PR.